### PR TITLE
Create task project and added project list view

### PR DESCRIPTION
**Pull request by GauravButola**

### CODE REVIEW

File: ng-app/app/scripts/app.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ angular.module('nereidProjectApp', [
   .config(function($stateProvider, $urlRouterProvider, $mdThemingProvider) {
     $urlRouterProvider
       .when('', '/')
+      .when('/', '/projects')
```
Comment (tarunbhardwaj) [inline ng-app/app/scripts/app.js]: '/projects/'

File: ng-app/app/scripts/controllers/task.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+      $scope.taskId = $state.params.taskId;
+      $scope.projectId = $state.params.projectId;
+
+      $scope.commentObj = {};
+
+      $scope.states = Task.states;
+
+      $scope.loadTask = function() {
+        Task.get($scope.projectId, $scope.taskId)
```
Comment (tarunbhardwaj) [inline ng-app/app/scripts/controllers/task.js]: Fetch by just using task id.

File: ng-app/app/scripts/app.js [hunk 3]
```diff
@@ -4,11 +4,13 @@ angular.module('nereidProjectApp', [
   'ui.router',
   'ngMaterial',
   'openlabs.angular-nereid-auth',
-  'cfp.hotkeys'
+  'cfp.hotkeys',
+  'infinite-scroll'
   ])
   .config(function($stateProvider, $urlRouterProvider, $mdThemingProvider) {
     $urlRouterProvider
       .when('', '/')
```
Comment (tarunbhardwaj) [inline ng-app/app/scripts/app.js:12]: do you really need this condition ?

Comment (GauravButola) [inline ng-app/app/scripts/app.js:12]: App will go to 404 without it

File: ng-app/app/scripts/directives/new-task.js [hunk 4]
```diff
@@ -3,18 +3,26 @@
...
           scope.openNewTaskModal = function() {
             $mdDialog.show({
               controller: function createTaskCtrl($scope, $mdDialog){
+
+                $scope.project = scope.getProject();
+                $scope.taskObj = {};
+
                 $scope.submit = function() {
-                  // TODO: handle form submit
+                  console.log($scope.taskObj);
```
Comment (tarunbhardwaj) [inline ng-app/app/scripts/directives/new-task.js]: ?

**Pull request merged**